api/v1alpha3/azurecluster_default.go

```diff
@@ -89,7 +89,7 @@ func (c *AzureCluster) setSubnetDefaults() {
 		cpSubnet.SecurityGroup.Name = generateControlPlaneSecurityGroupName(c.ObjectMeta.Name)
 	}
 	if cpSubnet.RouteTable.Name == "" {
-		cpSubnet.RouteTable.Name = generateRouteTableName(c.ObjectMeta.Name)
+		cpSubnet.RouteTable.Name = generateNodeRouteTableName(c.ObjectMeta.Name)
```

Comment:
this should not be set. The node route table is only associated with the node subnet.

Reply:
I though we couldn't leave it empty?

Reply:
ok, will make the change and test it out